src/App.jsx

```diff
@@ -14,70 +15,55 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [curGood, setCurGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+      <h1 className="title is-flex is-align-items-center">
+        {
+        curGood === ''
+          ? 'No goods selected'
+          : `${curGood} is selected`}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+        {curGood !== '' && (
```

Comment:
```suggestion
        {!!curGood && (
```

Reply:
це ж не  читабельно

Reply:
sorry I will not agree)

`!!` means that you transform your value into boolean, so if its not empty then the button will appear

actually you can even avoid using `!!` and write just `{curGood && (` because it will automatically transform to boolean